### PR TITLE
Notification with balance update (merge this first)

### PR DESCRIPTION
**Pull request by phy25**
- As designed, if user dismisses/deletes the notification, the balance will not be pushed again until next login attempt.
- A static BROADCAST RECEIVER has been added to catch notification deletion event below API 23. If there is anything wrong for the best practice, please let me know.

### CODE REVIEW

**Comment by LangleyChang** [comment]
Apologies but I'm really busy for some personal stuff recently and I'll review it several days later.

**Comment by phy25** [comment]
No problem, still coding...

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# BIT_NetCheckin2
+# BITion a.k.a BIT_NetCheckin2
```
Comment (LangleyChang) [inline README.md]: I don't understand this.

File: app/src/main/java/org/bitnp/netcheckin2/broadcast/NotifDeletedReceiver.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import org.bitnp.netcheckin2.service.LoginService;
+import org.bitnp.netcheckin2.util.NotifTools;
+
+/**
+ * Created by Phy25 on 7/1/2016.
+ */
+public class NotifDeletedReceiver extends BroadcastReceiver{
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        NotifTools.setNotif0Cleared(true);
```
Comment (LangleyChang) [inline app/src/main/java/org/bitnp/netcheckin2/broadcast/NotifDeletedReceiver.java]: If you just clear notifs, you don't have to use broadcast.

Comment (phy25) [inline app/src/main/java/org/bitnp/netcheckin2/broadcast/NotifDeletedReceiver.java]: Intended actions: When user "swipes" the notification, don't push balance
changes to notification bar anymore.

As geting current alive notification API is only on API level 23, I have to
use broadcast to receive the event of user cancellation. If there are
better apporach, just let me know.

Comment (LangleyChang) [inline app/src/main/java/org/bitnp/netcheckin2/broadcast/NotifDeletedReceiver.java]: It's okay.

File: app/src/main/java/org/bitnp/netcheckin2/util/NotifTools.java [hunk 3]
```diff
@@ -20,6 +22,8 @@
 public class NotifTools {
     private static NotificationManager mNotificationManager;
     private static NotifTools instance;
+    private static boolean notif0Cleared = false;
+    private static Bitmap logoBM;
 
     private NotifTools(){}
```
Comment (LangleyChang) [inline app/src/main/java/org/bitnp/netcheckin2/util/NotifTools.java]: why not set all the methods `static` instead of singleton pattern.

**Pull request merged**